pkg/client/connector/auth/login.go

```diff
@@ -108,19 +107,6 @@ func NewLoginExecutor(
 	return ret
 }
 
-// EnsureLoggedIn will check if the user is logged in and if not initiate the login flow.
-func EnsureLoggedIn(ctx context.Context, executor LoginExecutor) (connector.LoginResult_Code, error) {
```

Comment:
Okay so I understand this issue bc EnsureLoggedIn would see that the token file existed and would return OLD_LOGIN_REUSED, regardless of whether that token was expired or not.

Reply:
Yeah.